src/Sylius/Bundle/AttributeBundle/Form/EventSubscriber/BuildAttributeValueFormSubscriber.php

```diff
@@ -86,6 +95,16 @@ private function addValueField(FormInterface $form, AttributeInterface $attribut
     {
         $options = ['auto_initialize' => false, 'label' => $attribute->getName()];
 
-        $form->add('value', 'sylius_attribute_type_'.$attribute->getType(), $options);
+        if ($attribute->isValueTranslatable()) {
+            $form->add('translations', 'a2lix_translationsForms', [
+                'form_type' => sprintf('sylius_%s_attribute_value_translation', $this->subjectName),
+                'label' => 'sylius.form.value_translation.presentation',
+                'form_options' => [
+                    'valueTranslationType' => $attribute->getType(),
+                ],
+            ]);
+        } else {
+            $form->add('value', 'sylius_attribute_type_'.$attribute->getType(), $options);
+        }
```

Comment:
With good design of service I mentioned [here](https://github.com/Sylius/Sylius/pull/4512/files#r56236922), I think it could be also used here to prevent this nasty ``else`` statement and code duplication ;)